transcom/mymove
# B-20080 INT (Part #2) - Add Marine Corps to Move Search Filter

## [B-20080](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20080)

## Summary

Adds Marine Corps to the branch filter in the move search results. 

> [!NOTE]  
> The TOO and TIO queues' branch filters also do not include Marine Corps, this appears to be intentional as only USMC GBLOC TOOs and TIOs will be shown Marine Corps moves and the filter is not shown for them since there would be nothing to filter.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the office app
2. Login as a user that can search moves (I think that includes all roles now).
3. Search for moves with any query.
4. Verify the branch filter includes an option for Marine Corps.
5. Optionally ensure the filter works by adding a Marine Corps customer and move.


### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
